src/components/InjectedComponents/PostDialog.tsx

```diff
@@ -342,30 +345,25 @@ export function PostDialog(props: PostDialogProps) {
             setPostBoxContent(makeTypedMessage(''))
             setCurrentShareTarget([])
             getActivatedUI().typedMessageMetadata.value = new Map()
-        }, []),
+        }, [setOpen]),
```

Comment:
Any problems does this code fixed?

Reply:
very likely these were added by eslint...

Reply:
It because now `open, setOpen` can be provided from props and no longer to be a local hook.